bindings/python/MANIFEST.in

```diff
@@ -2,13 +2,19 @@ include README.rst
 include LICENSE
 include build-libbson.sh
 include release.sh
```

Comment:
Should we remove the release script here?

Reply:
Sounds good